### PR TITLE
DVO-5724 -> master | Implement 'up' functionality for gull

### PR DESCRIPTION
**Pull request by tim-kretschmer-c2fo**
[JIRA Link](https://honeycomb.jira.com/browse/DVO-5724)
##### Issuer

@tim-kretschmer-c2fo - Tim Kretschmer
##### Code Review + Discussion Complete

If any user has commented on this PR, but has not checked his or her box below, then the PR cannot be merged.

Team Lead
- [ ] Team Lead Reviewed

Team
- [ ] @achura - Adam Chura
- [ ] @thomascarrollc2fo - Thomas Carroll
- [ ] @brockwood - Bryan Rockwood

### CODE REVIEW

**Comment by tim-kretschmer-c2fo** [comment]
Note that my test dataset for this has been the c2fo node baseline. There's a TODO to implement migrations based on non-environmentally specific configs.

File: script/etcd/write-simple.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
```
Comment (brockwood) [inline script/etcd/write-simple.sh:1]: What is the purpose of this (besides the literary refresher :+1:)?

File: source/lib/gull/etcd.go [hunk 2]
```diff
@@ -2,40 +2,58 @@ package gull
...
-		Method: "PUT",
-		Uri:    storageUrl,
-		Body:   value,
+		Method:      "PUT",
+		Uri:         storageUrl,
+		Body:        value,
+		ContentType: "application/x-www-form-urlencoded",
 	}.Do()
 	if response != nil {
 		defer func() { _ = response.Body.Close() }()
```
Comment (brockwood) [inline source/lib/gull/etcd.go:36]: Why wrap this in a closure?

File: script/etcd/write-simple.sh (hunk 1 above)
Comment (tim-kretschmer-c2fo) [inline script/etcd/write-simple.sh:1]: I was testing out whether or not reads from etcd were working. Wanted to test it against a known set of written data.

File: source/lib/gull/etcd.go (hunk 2 above)
Comment (tim-kretschmer-c2fo) [inline source/lib/gull/etcd.go:36]: IIRC it was a habit I got into after running either govet or errcheck against the c2fo-go baseline. Speaking of that, I'm going to open a ticket to see if we can automatically run those two in travis.

**Pull request merged**